packages/earth-admin/src/pages-client/dashboards/details.tsx

```diff
@@ -64,9 +64,14 @@ export function DashboardDetail(props: IProps) {
   const query = merge(DASHBOARD_DETAIL_QUERY, { group: selectedGroup });
   const cacheKey = generateCacheKey(`dashboards/${page}`, query);
 
-  const { data, error, mutate } = useSWR(cacheKey, () =>
-    DashboardsService.getDashboard(page, query).then((res: any) => res.data)
-  );
+  const fetcher = () =>
+    DashboardsService.getDashboard(page, query).then((response: any) => response.data);
+  const setter = (data) =>
+    DashboardsService.handleDashboardForm(false, data, id, query).then(
+      (response: any) => response.data
+    );
+
+  const { data, error, revalidate } = useSWR(cacheKey, fetcher);
```

Comment:
useSWR returns a mutate function that is already bound to that cacheKey. Makes things a bit easier.
Can you update all the pages to this simpler version?

```suggestion
  const { data, error, mutate } = useSWR(cacheKey, fetcher);
```